servers/rendering/renderer_rd/effects/copy_effects.h

```diff
@@ -172,11 +172,11 @@ class CopyEffects {
 		COPY_TO_FB_COPY,
 		COPY_TO_FB_COPY_PANORAMA_TO_DP,
 		COPY_TO_FB_COPY2,
+		COPY_TO_FB_SET_COLOR,
 
```

Comment:
I would add a comment here to clarify that the variants below are optional and should come last in the enum.

Reply:
Done, let me know if you agree with the text :)